backend/src/memberships/entities/membership.entity.ts

```diff
@@ -26,12 +26,14 @@ export class Membership {
   @ManyToOne(() => Channel)
```

Comment:
```suggestion
  @ManyToOne(() => Channel, { onDelete: 'CASCADE' })
```

Otherwise a channel cannot be deleted if a membership exists on that channel